app/src/main/res/layout/layout_tokens_item.xml

```diff
@@ -1,12 +1,38 @@
 <?xml version="1.0" encoding="utf-8"?>
-<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/layout_tokens_item_name"
+    android:id="@+id/layout_tokens_item_root"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:background="@drawable/selectable_background"
-    android:ellipsize="end"
-    android:maxLines="2"
-    android:padding="16dp"
-    android:textSize="16sp"
-    tools:text="Gnosis" />
+    android:gravity="center_vertical"
+    android:padding="16dp">
+
+    <android.support.v7.widget.AppCompatTextView
+        android:id="@+id/layout_tokens_item_symbol"
+        style="@style/Title"
+        android:layout_width="@dimen/action_circle"
+        android:layout_height="@dimen/action_circle"
+        android:background="@drawable/action_circle_background"
+        android:ellipsize="end"
+        android:gravity="center"
+        android:maxLines="1"
+        android:paddingLeft="4dp"
+        android:paddingRight="4dp"
+        app:autoSizeMaxTextSize="14sp"
+        app:autoSizeMinTextSize="8sp"
+        app:autoSizeTextType="uniform"
+        tools:text="GNO" />
+
+    <TextView
+        android:id="@+id/layout_tokens_item_name"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="12dp"
+        android:ellipsize="end"
+        android:maxLines="2"
+        android:textColor="@color/gnosis_dark_blue"
+        android:textSize="15sp"
```

Comment:
We should use some unified text sizes (we have 16sp in the settings screen, which is basically the same)